modules/distr-tracing-rn-technology-preview.adoc

```diff
@@ -18,6 +18,25 @@ Technology Preview features are not supported with Red Hat production service le
 For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
 
+== {DTProductName} 2.8.0 Technology Preview
+
+This release introduces support for {TempoName} as a Technology Preview feature for {DTProductName}.
+The feature uses {TempoName} version 0.1.0 and version 2.0.1 of the upstream {TempoShortName} components.
```

Comment:
On first read, it sounds like {TempoName} uses version 0.1.0 and 2.0.1 ... it might read better if it says something like: "The feature uses version 0.1.0 of {TempoName} and version 2.0.1 of the upstream {TempoShortName} components."

Reply:
> On first read, it sounds like {TempoName} uses version 0.1.0 and 2.0.1 ... it might read better if it says something like: "The feature uses version 0.1.0 of {TempoName} and version 2.0.1 of the upstream {TempoShortName} components."

Agree. I'll make the change. Good catch!